bootstrap/common/sync_third_party.sh

```diff
@@ -76,7 +81,12 @@ function sync_third_party() {
 
   # Move final result to $target_path.
   rm -rf $target_path
-  mv $extracted_path $target_path
+  if is_windows_platform; then
+    cp -R $extracted_path $target_path
```

Comment:
This will be very slow. Do we really have to copy it?

Reply:
um.. when drive name is different with origin path and target path, mv command was needed permission even though has administrator permission.
when drive name is same, it was not needed.